addOns/llm/src/main/javahelp/org/zaproxy/addon/llm/resources/help/map.jhm

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+	<mapID target="simple-icon" url="contents/images/cake.png" />
```

Comment:
all instances of `simple` -> `llm`. Will need a new icon - I'll look for something suitable 😁 Probably dont need the `about.html` page?